app/controllers/sign_workflow_controller.rb

```diff
@@ -42,8 +42,19 @@ def decline
 
   private
 
-  def succeeded
-    redirect_back(fallback_location: sign, notice: success_messsage)
+  def succeeded(redirect_path: nil)
```

Comment:
@hopkincame are there any situations where `redirect_path` _isn't_ `redirect_location` here?

It seems like you could just use `redirect_to redirect_location, notice: success_message` - you're not returning `nil` from `redirect_location`, so is the `else` ever actually executed?

Also, you might consider naming `redirect_location` something that indicates it's purpose, like Devise does with `after_sign_in_path_for`, `after_sign_out_path` etc - `after_succeeded_path_for(sign)`?

Reply:
This was done to avoid modifying the controller actions that were not broken. Everything that is not unpublish or decline will take the else route and do the old redirect back. If the redirect path is accurate for the other controller actions though, then I can update this so that all of them use the redirect to.